war/.jshintrc.json

```diff
@@ -0,0 +1,11 @@
+{
+  "curly": true,
```

Comment:
I guess it can probably be found easily in the documentation, but maybe a comment explaining what each option means would be helpful. This is a quite big project and not everybody could be knowledgeable about this.

Reply:
this is a json file, is it possible to add comments?

Reply:
You cannot leave comments in these files.

This config file belongs to jshint, it's the default one. I'll remove it as I'll try to replace jshint with eslint. On eslint, I'm partial too looking for a standard config, so that developers don't need to worry about the reasoning for each specific rule.